### PR TITLE
Added tags

### PR DESCRIPTION
**Pull request by yehudahrrs**
This is my first time doing anything in JS, hope it works

### CODE REVIEW

File: bot_modules/search_commands/tags.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+const Discord = require('discord.js')
+const Utils = require('../../utils')
+
+/**
+ * Handle the tag command
+ *
+ * @param {Discord.Message} msg The original message send by the user
+ * @param {Array} args The list of arguments passed to the command
+ */
+exports.handleTagCommand = async (msg, args) => {
```
Comment (Heath123) [inline bot_modules/search_commands/tags.js:11]: Where is this function used?

File: bot_modules/search_commands/tags.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  tags.forEach((tag) => {
+    if (tag.name.toLowerCase().startsWith(query.toLowerCase())) {
+      foundTag = true
+
+      sendTagEmbed(msg, tag)
+    }
+  })
+
+  // If we haven't found a tag yet use an
+  // algorythm to check for the most similar tag
```
Comment (Heath123) [inline bot_modules/search_commands/tags.js:41]: This is misspelled...

Comment (Heath123) [reply bot_modules/search_commands/tags.js:41]: Also should we use string similarity or only accept exact matches?

File: bot_modules/search_commands/tags.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      foundTag = true
+
+      sendTagEmbed(msg, tag)
+    }
+  }
+
+  // Send a message if we dont know what tag
+  if (!foundTag) {
+    embed.setTitle('Unknown tag')
+    embed.setDescription('That tag is not on our GitHub so it is untested!')
```
Comment (Heath123) [inline bot_modules/search_commands/tags.js:58]: Uh, I think this message was copied from the providers thing, a tag can't really be untested

**Review by Heath123 — CHANGES_REQUESTED** [review]
Seems mostly good I think but is it tested? I'll look in more detail later

File: bot_modules/search_commands/tags.js (hunk 2 above)
Comment (rtm516) [reply bot_modules/search_commands/tags.js:41]: Only exact matches makes sense for tags

File: bot_modules/search_commands/tags.js [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  msg.channel.send(embed)
+}
+
+/**
+ * Get the tags from the github page
+ * and parse + format the results
+ *
+ * @returns {Array} The list of tag objects with name, url, instructions and category
+ */
+async function getTags () {
```
Comment (rtm516) [inline bot_modules/search_commands/tags.js:91]: This function shouldn't work the same as the others, really I think this system needs to work of a repo or folder not a single wiki page. Like the Fabric Bot https://github.com/FabricMC/community/tree/main/tags

**Pull request closed without merging**